concrete/single_pages/dashboard/system/mail/method.php

```diff
@@ -83,19 +94,29 @@
 
     <div class="ccm-dashboard-form-actions-wrapper">
         <div class="ccm-dashboard-form-actions">
-            <a href="<?= $this->action('test') ?>" class="btn btn-secondary float-start"><?= t('Test Settings') ?></a>
+            <a href="<?= $view->action('test') ?>" class="btn btn-secondary float-start"><?= t('Test Settings') ?></a>
```

Comment:
`$this` and `$view` contains exactly the same object (an instance of `Concrete\Core\Page\View\PageView`, but IMHO `$view` is cleaner than `$this`).